config/manifests/bases/orchestrator-operator.clusterserviceversion.yaml

```diff
@@ -71,13 +71,13 @@ spec:
   - Backstage
   links:
   - name: Orchestrator Operator
-    url: https://github.com/parodos-dev/orchestrator-helm-operator
+    url: https://github.com/rhdhorchestrator/orchestrator-helm-operator
   maintainers:
   - email: jgil@redhat.com
     name: Red Hat
-  maturity: alpha
+  maturity: stable
```

Comment:
We're not yet GA, I would not think we're `stable` yet 😄  . We might be using the stable channel for the operator, but I consider that to be just a way to split between new development(fast/alpha) vs existing releases.

Reply:
Got it. I had read the field indicates the maturity level, then thought we were stable forgetting we are not GA yet. LOL. My bad. Will update it.

Reply:
Done